### DISCUSSION TITLE
What this message means? "'NoneType' object has no attribute 'tb_lineno'@line 2528"

### DISCUSSION THREAD

**Discussion by Rillrules** [artisan-roaster-scope/artisan#733] [Q&A]
Hello.
I'm using Artisan with Center306 Thermometer. I have used with no problem, but now Artisan can't read data with following message.

CENTER306temperature() 'NoneType' object has no attribute 'tb_lineno'@line 2528

Thermometer seems to works and recognized well.. Is there anybody who know that message means? And how can I solve this problem?
Thanks in advance.

**Comment by poundy** [comment]
I'd suggest the first check is whether the port that the meter presents to the computer has changed for some reason (using a different USB socket will do it). I'm guessing, but I always suspect that something that is easily overlooked is the culprit. If on Windows, check Device Manager to see what the COM port shows up when you connect the meter, and confirm Artisan setup is matching that.

**Reply by Rillrules** [reply]
Thank you for your advise, but I checked that COM port of Device manager and Artisan setup are matched each other. I tested all of USB port of my computer and it was same.

**Comment by MAKOMO** [comment]
Did you upgrade your SiLab CP201x driver?

**Reply by Rillrules** [reply]
Yes. I updated it but it was same. I think it is problem of thermometer.
